dataflow/src/main/java/org/checkerframework/dataflow/cfg/node/AbstractNodeVisitor.java

```diff
@@ -159,6 +159,7 @@ public R visitBitwiseXor(BitwiseXorNode n, P p) {
     }
 
     // Compound assignments
+    @Deprecated()
```

Comment:
Just use `@Deprecated`, without the empty parenthesis.

Reply:
fixed.